tests/providers/common/sql/hooks/test_sql.py

```diff
@@ -251,3 +252,12 @@ def test_make_common_data_structure_no_deprecated_method(self):
     def test_placeholder_config_from_extra(self):
         dbapi_hook = mock_hook(DbApiHook, conn_params={"extra": {"placeholder": "?"}})
         assert dbapi_hook.placeholder == "?"
+
+    def test_suppress_and_warn_when_raised_exception_is_suppressed(self):
```

Comment:
If we limit it to **only** jdbc - that one needs to be moved too :D

Reply:
Yes I thought I reverted that one as I've adapted it in the wrong branch, it's reverted now